platform/tasks-platform-api/resources/messages/TaskApiBundle.properties

```diff
@@ -11,6 +11,7 @@ server.u.rl=Server U&RL:
 sh.are.url=Sh&are URL
 te.st=Te&st
 use=Use...
+use.bearer.token=Use Personal Access &Token
```

Comment:
Another bikeshedding thing is that according to our UI guidelines, checkbox labels should use sentence-style capitalization, i.e. "Use personal access token". "Login Anonymously" actually violates it, I will update it.